### ISSUE TITLE
Torrent list resets when changing 'Table Resize Style' settings

### ISSUE THREAD

**Issue by valdotdev** [Tympanix/Electorrent#8]
I'm not sure if this was intentional, but whenever I change the 'Table Resize Style' settings and saving, the torrent list resets (empty). I then need to manually refresh/reload the app to restore the list.

**Comment by Tympanix** [comment]
I have not been able to see nor experience this bug throughout my testing. Closing for now until someone finds a way to reliably reproduce this bug.

**Comment by valdotdev** [comment]
Still happening on ver 1.2.0 though. And since the option to 'Reload' the app was removed from this version, I need to close the app first then launch it again. I suggest bringing back the reload option in the menu bar at least for now :)

So here's exactly what I did:
Launch Electorrent>Open Preferences>General Settings>Change 'Table Resize Style' setting (eg. from 'Fixed' to 'Basic')>Click on 'Save'. To be honest, I don't have a clue what this settings does :D

After that, the whole table/list becomes empty so I need to reload the application.

Maybe this could help. I'm using the latest stable Windows x64 build of qBittorrent (v3.3.6) running on Windows 10 x64. And I use the Mac version of Electorrent runnning on OS X 10.11.6

So for now, I suggest bringing back the reload option. Of course, you don't really need to since this this is just a minor issue and I won't be playing around with the settings very much. It was just a thing I noticed when I was browsing the settings.
